Lectures/Week-02-ExpressionsAndLoops.ipynb

```diff
@@ -176,7 +176,7 @@
     "\n",
     "# More context about what I'll cover in lecture\n",
     "\n",
-    "My lectures are not meant to be 1:1 what is in the book or perfectly and only what you need for the homework.  There will be many places where I'll have a clearly direct example for the homework, but I'm also preparing you for the midterm.  All the information you'll need for the homework will be in the book (if it is a book problem).  My goal with lecture is to give you an alternative presentation reinforce and explore what you have been learning in your readings.  \n",
+    "My lectures are not meant to be 1:1 what is in the book or perfectly and only what you need for the homework.  There will be many places where I'll have a clearly direct example for the homework, but I'm also preparing you for the midterm.  All the information you'll need for the homework will be in the book (if it is a book problem).  My goal with lecture is to give you an alternative presentation to reinforce and explore what you have been learning in your readings.  \n",
```

Comment:
it's possible this was meant as a sequence of three things, so might not actually be a typo. Can easily rebase out of this PR if so.